Taggers/python/flashggTags_cff.py

```diff
@@ -176,12 +181,16 @@
                                        GenParticleTag=cms.InputTag( "flashggPrunedGenParticles" ),
 				       rhoTag = cms.InputTag('fixedGridRhoFastjetAll'),
                                        MVAweightfile = cms.FileInPath("flashgg/Taggers/data/TMVAClassification_BDT_training_v2.json.weights.xml"),
-                                       leadPhoOverMassThreshold = cms.double(0.33),
-                                       subleadPhoOverMassThreshold = cms.double(0.25),
-                                       MVAThreshold = cms.vdouble(0.6, 0.3),
-                                       PhoMVAThreshold = cms.double(-0.2), 
+				       topTaggerXMLfile = cms.FileInPath("flashgg/Taggers/data/resTop_xgb_csv_order_deepCTag.xml"),
```

Comment:
same here

Reply:
Same here as well: single training for all 3 years.